packages/react-vapor/src/components/datePicker/DatePickerDropdown.tsx

```diff
@@ -117,30 +116,21 @@ export class DatePickerDropdown extends React.Component<IDatePickerDropdownProps
         let label: string = this.props.label;
         let toLabel: JSX.Element = null;
         let labelSecondPart: string = '';
+        const disabled: boolean = this.props.readonly;

```

Comment:
I don't see the added value of doing this so I would tend to use the `readonly` prop directly everywhere.